Source/ACE/Entity/WorldObject.cs

```diff
@@ -193,166 +188,199 @@ public ContainerType ContainerType
-            set { AceObject.TargetTypeId = value; }
+            get { return AceObject.GetIntProperty(PropertyInt.TargetType); }
+            set { AceObject.SetIntProperty(PropertyInt.TargetType, value); }
         }
 
         public UiEffects? UiEffects
         {
-            get { return (UiEffects?)AceObject.UiEffects; }
-            set { AceObject.UiEffects = (uint?)value; }
+            get { return (UiEffects?)AceObject.GetIntProperty(PropertyInt.UiEffects); }
+            set { AceObject.SetIntProperty(PropertyInt.UiEffects, (uint?)value); }
         }
 
         public CombatUse? CombatUse
         {
-            get { return (CombatUse?)AceObject.CombatUse; }
-            set { AceObject.CombatUse = (byte?)value; }
+            get { return (CombatUse?)AceObject.GetIntProperty(PropertyInt.CombatUse); }
+            set { AceObject.SetIntProperty(PropertyInt.CombatUse, (byte?)value); }
         }
 
         public MotionStance? DefaultCombatStyle
         {
-            get { return (MotionStance?)AceObject.DefaultCombatStyle; }
-            set { AceObject.DefaultCombatStyle = (uint?)value; }
+            get { return (MotionStance?)AceObject.GetIntProperty(PropertyInt.DefaultCombatStyle); }
+            set { AceObject.SetIntProperty(PropertyInt.DefaultCombatStyle, (uint?)value); }
         }
 
         /// <summary>
         /// This is used to indicate the number of uses remaining.  Example 32 uses left out of 50 (MaxStructure)
         /// </summary>
         public ushort? Structure
         {
-            get { return AceObject.Structure; }
-            set { AceObject.Structure = value; }
+            get { return (ushort?)AceObject.GetIntProperty(PropertyInt.Structure); }
+            set { AceObject.SetIntProperty(PropertyInt.Structure, value); }
         }
 
         /// <summary>
         /// Use Limit - example 50 use healing kit
         /// </summary>
         public ushort? MaxStructure
         {
-            get { return AceObject.MaxStructure; }
-            set { AceObject.MaxStructure = value; }
+            get { return (ushort?)AceObject.GetIntProperty(PropertyInt.MaxStructure); }
+            set { AceObject.SetIntProperty(PropertyInt.MaxStructure, value); }
         }
 
         public ushort? StackSize
         {
-            get { return AceObject.StackSize; }
-            set { AceObject.StackSize = value; }
+            get { return (ushort?)AceObject.GetIntProperty(PropertyInt.StackSize); }
+            set { AceObject.SetIntProperty(PropertyInt.StackSize, value); }
         }
 
         public ushort? MaxStackSize
         {
-            get { return AceObject.MaxStackSize; }
-            set { AceObject.MaxStackSize = value; }
+            get { return (ushort?)AceObject.GetIntProperty(PropertyInt.MaxStackSize); }
+            set { AceObject.SetIntProperty(PropertyInt.MaxStackSize, value); }
         }
 
-        public uint? ContainerId
+        public uint? Container
```

Comment:
"Container" is an object, this is an identifier.  ContainerId is strongly preferred.